guides/doc-Configuring_Smart_Proxies_with_a_Load_Balancer/master.adoc

```diff
@@ -0,0 +1,46 @@
+:imagesdir: images
+:numbered:
+ifeval::["{build}" != "satellite"]
+:toc:
+:toc-placement: left
+endif::[]
+
+include::common/attributes.adoc[]
+
+= Configuring {SmartProxies} with a Load Balancer
```

Comment:
Oh this is the actual book name, then probably `doc-Configuring_Proxy_with_Load_Balancer` although this is quite long to my taste. Technically, you can have a single instance behind a load balancer, although plural is probably more appropriate and I am not against it.

Reply:
If we do embrace short directory names (#194) I'd be happy to trim this directory name down to `doc-Configuring_Load_Balancer`.

Reply:
`doc-Configuring_Load_Balancer` sounds good to me. Would you make this change in this PR @tahliar ?

Reply:
Yeah, since you approve of the suggestion, I'll do it in this PR.